### DISCUSSION TITLE
Docker Container?

### DISCUSSION THREAD

**Discussion by JohnHind** [monkeymademe/CamUI#50] [Ideas]
Just wondered if anybody had put this in a Docker container or considered this? Seems like an ideal candidate for this approach.
By the way, I found this through a Jeff Geerling Youtube. I've been looking for something like this for ages and was surprised the Pi community were not on it sooner!

**Comment by monkeymademe** [comment]
Its on my bucket list but I heard that exposing the camera to a container is 'easy' but difficult to get working with picamera2.... I might be wrong though. At the moment I am focused on getting the app working
